advocacy_docs/partner_docs/VeritasGuide/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+NetBackup for PostgreSQL Agent extends the capabilities of NetBackup to include backup and restore of PostgreSQL databases. If a NetBackup environment is operational within an organization, then users can backup and restore EDB Postgres Advanced and EDB Postgres Extended database servers with the help of NetBackup for PostgreSQL Agent. 
```

Comment:
I'd like to see this be more detailed, we should look at the NetBackup related documents in the Veritas partner folder for more information we can add to the solution summary.

Reply:
See response to Scott's comment